docs/op-stack/src/docs/security/pause.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Pause and Unpause the Bridge
+lang: en-US
+---
+
+
+## Why do it?
+
+The bridge allows movement between blockchains. 
+If the security of a blockchain is compromised, pausing bridge withdrawals will mitigate damage until the issue is resolved.
+
+## Who can do it?
+
+[`OptimismPortal`](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/contracts/L1/OptimismPortal.sol) has an immutable `GUARDIAN`. 
+That address can call [`pause`](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/contracts/L1/OptimismPortal.sol#L166-L170) and [`unpause`](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/contracts/L1/OptimismPortal.sol#L175-L179).
+
+
+### Changing the guardian
+
+The guardian created by the setup script is the admin account.
+This is sufficient for testing, but for a production system you would want the guardian to be a multisig with trusted security council.
+
+The guardian is immutable, there is no way to change it while using the same contract.
+Luckily, it isn't supposed to be called directly, but through a proxy.
+[You can tell the proxy to go to a new contract](../build/tutorials/new-precomp.md).
```

Comment:
Since this contract is on L1, you need to upgrade on L1 and not on L2. Predeploy stuff is for L2.